sonic_platform_base/sonic_xcvr/fields/consts.py

```diff
@@ -142,7 +142,8 @@
 
 # CMIS
 
-ADMIN_INFO_FIELD = "AdminInfo"
+ADMIN_INFO_FIELD0 = "AdminInfoLowerPage"
```

Comment:
I think we should keep `ADMIN_INFO_FIELD` and use `ADVERTISING_FIELD` for the page 01h fields, as these are the names used in the spec.